yarn.lock

```diff
@@ -1252,7 +1252,7 @@
   dependencies:
     regenerator-runtime "^0.13.4"
 
-"@babel/runtime@^7.10.1":
+"@babel/runtime@^7.10.1", "@babel/runtime@^7.11.2":
```

Comment:
why did you update @babel?

Reply:
that's also because of the merge. the master-branch uses 7.11.2 [see here](https://github.com/wepublish/wepublish/blob/2968cf711d39c58dcec51916a4803056f3528f17/yarn.lock#L1255)

Reply:
Sorry. I got confused. I didn't know that github highlighted those changes.